test.js

```diff
@@ -1,3 +1,4 @@
+import {execFile} from 'child_process';
 import test from 'ava';
 import camelcaseKeys from '.';
 
```

Comment:
```suggestion
import {promising} from 'node:util';

const execFilePromise = promsify(execFile);
```

Reply:
Good idea 👍

I pushed a commit with this change. We can't use `node:util` on Node 12.x.